_i18n/en.yml

```diff
@@ -141,14 +140,42 @@ sponsorships:
   macstadium: MacStadium sponsors a dedicated Mac Mini that is used for Mac-related development.
 
 team:
-  core: Core
-  developers: Developers
-  developers_para1: The Monero Project has had well over 500 contributors over the life of the project. For a complete list, please see the
-  developers_para2: OpenHub contributors page.
-  developers_para3: Below you'll find some individuals that have gone above and beyond for Monero.
-  community: Community
+  introduction: Workgroups are formed by community members who join forces to achieve a common goal. They have different structures and goals. On this page we list some of the more active workgroups and some tips to get you started, if you wish to contribute to any of them.
+  core: Core Team
+  community: Community Workgroup
   mrl: Research Lab
-  thanks: Special Thanks
+  descr: Description
+  start: Where to start
+  contacts: Chatrooms and Contacts
+  chat: "Chat:"
+  website: "Website:"
+  core_descr: The members of this workgroup are always 7. They are the stewards of the community and they manage and maintain the Monero codebase and infrastructure. The Core Team act as mediators, weighting consensus in the community and taking decisions according to community's will.
```

Comment:
core_descr: Stewards of the community, the Core Team manages and maintains infrastructure where centralization cannot be avoided (domains, website, github repositories...). Its members act as mediators as they see fit and aim to weigh consensus in the community, perpetuating the quality interactions that made Monero the successful project it is today. The Core Team does not and has never received any financial compensation.

Reply:
Note this text was agreed with the others

Reply:
Updated